docs/rfcs/2025-01-20-member-mentions.md

```diff
@@ -0,0 +1,39 @@
+# Member mentions
+
+## Problem
+
+Mention members in the messages.
+
+## Solution
+
+Message text can include references to the shared member IDs in base64 format:
+
+```
+Hello @'abcd1234'
+```
+
+In addition, the message JSON will include the array of mentions, as objects with property `memberId`. This is to ensure the intent and that the fragments of text are treated as mentions.
+
+Using an immutable `memberId` would prevent any race conditions and duplicate display names. The receiving client would show a local view name (display name or an alias), and would open a correct member card when mention is tapped.
+
+There should be a reasonable limit on the number of mentions in a group, e.g. 3. This is to prevent abuse, expensive processing both in the client and in super-peers that would have to forward member profiles if they were not forwarded before. This limit has to be enforced both on sending and receiving ends.
```

Comment:
how will mention look in UI in chat item's text? possibly it's not necessary to render avatar (as tapping it already opens member info page), so sending/forwarding mentions with name included should be sufficient without additionally sending profiles, even if receiving client doesn't have that member's profile.

Reply:
Just a highlighted name. For forwards - not even highlighted, as they don't refer to the current members.